src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html

```diff
@@ -905,10 +905,7 @@ <h5 class="col-sm-2">
 
 
     
-        <div id="statusMessage" class="alert alert-warning">
-            The feedback session has been updated.
-        </div>
-        <script type="text/javascript" src="/js/statusMessage.js"></script>
+        <div id="statusMessage" style="display: none;"></div>
```

Comment:
Ok, then won't that mean that the page should be refreshed before verifying <code>instructorFeedbackEditSuccess.html</code> too?